components/mpas-ocean/cime_config/config_compsets.xml

```diff
@@ -72,6 +101,11 @@
     <lname>2000_DATM%JRA-1p5_SLND_MPASSI%DIB_MPASO%IBDISMFDATMFORCED_DROF%JRA-1p5-AIS0ROF_SGLC_SWAV</lname>
   </compset>
 
+  <compset>
+    <alias>GMPAS-JRA1p5-DIB-PISMF-TMIX</alias>
+    <lname>2000_DATM%JRA-1p5_SLND_MPASSI%DIB_MPASO%IBPISMFDATMFORCEDTMIX_DROF%JRA-1p5-AIS0ROF_SGLC_SWAV</lname>
+  </compset>
+
```

Comment:
@irenavankova, is this the configuration you're actually using for your runs? If so, let's keep this particular short name for now and remove all the other `-TMIX` short names.

Reply:
I kept this one and deleted all the rest